variables.tf

```diff
@@ -160,6 +160,24 @@ variable "enable_nat_gateway" {
   default     = true
 }
 
+variable "accept_vpc_endpoint_connection_requests" {
```

Comment:
`vpc_endpoint_service_accept_connection_requests` to be consistent with `vpc_endpoint_service_allowed_principals`